### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Mehul2410**
Added my portfolio and github id as suggested

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -140,6 +140,25 @@ <h1 class="title-font sm:text-2xl text-xl font-medium text-white mb-3">Portfolio
...
+                        <a class="text-blue-400 inline-flex items-center"
+                            href="http://mehul.vercel.app/">Learn More
+                            <svg class="w-4 h-4 ml-2" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2"
+                                fill="none" stroke-linecap="round" stroke-linejoin="round">
+                                <path d="M5 12h14"></path>
+                                <path d="M12 5l7 7-7 7"></path>
+                            </svg>
+                        </a>
+                    </div>
+                </div>
```
Comment (cmulay) [inline index.html:161]: LGTM !

File: index.html [hunk 2]
```diff
@@ -163,4 +182,4 @@ <h1 class="title-font sm:text-2xl text-xl font-medium text-white mb-3">Portfolio
     </footer>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (cmulay) [inline index.html:185]: Portfolio is to be uploaded to a folder with your username.

**Review by cmulay — CHANGES_REQUESTED** [review]
Missing portfolio source code directory.

File: index.html (hunk 2 above)
Comment (Mehul2410) [reply index.html:185]: Done

**Review by cmulay — APPROVED** [review]
LGTM, Changes made !

**Pull request merged**